RELEASING.md

```diff
@@ -0,0 +1,25 @@
+# Notes for releasing
+
+## Angular version
+
+For Bazel support downstream, the version of the @angular/bazel npm package
+in `package.json` and `tools/npm/package.json` must match the version of the
+`@angular` `http_archive` in `WORKSPACE`. For this reason, `@angular/bazel`
+should be pinned to a specific version in both `package.json` files. If updating
+the version of the angular http_archive, ensure sure that all 3 versions line up.
```

Comment:
This needs to just be an automatic check in our release script, otherwise it will 100% get broken

Reply:
I think we should chat about that `tools/npm` file. Based on some testing it's just needed because the Angular Bazel rules fall back to `@npm` for the compiler binaries etc.

RxJS apparently uses their own workspace now, and the comment about why we need this (just for rxjs) is no longer up-to date I think.

Reply:
It's still needed for rxjs since it uses ts_library in there which depends on the `@npm` workspace. An rxjs release with a named UMD bundle is in the works (https://github.com/ReactiveX/rxjs/pull/4309). When that lands and makes it into a release then the build can be updated to no longer build RXJS from source.